packages/core/src/browser/tree/tree-widget.ts

```diff
@@ -329,10 +329,26 @@ export class TreeWidget extends VirtualWidget implements StatefulWidget {
     }
 
     storeState(): object {
-        return this.deflateForStorage(this.model.root!);
+        if (this.model.root) {
+            return this.deflateForStorage(this.model.root);
```

Comment:
could we return 
```
if (this.model.root)
   return {
      root: this.deflateForStorage(this.model.root)
   }
else
   return {}
```
and then check for existence of `root` property in restoreState. (instead of the `isEmptyObject`contract)